service/src/settings.h

```diff
@@ -74,23 +83,33 @@ class Settings : public QObject
     int highLimit = 70;
     QString lowAlertFile = "/usr/share/sounds/jolla-ambient/stereo/general_warning.wav";
     QString highAlertFile = "/usr/share/sounds/jolla-ambient/stereo/positive_confirmation.wav";
+    QString healthAlertFile = "/usr/share/sounds/jolla-ambient/stereo/battery_low.wav";
```

Comment:
I don't think we should use system "battery low" sound for health, even more so because "general warning" is used for battery low... Yeah. I'll switch them post merge.